### ISSUE TITLE
Upload shortcode within WooCommerce '/my-account/edit-account/' page not working properly.

### ISSUE THREAD

**Issue by Marcquito** [strangerstudios/Basic-User-Avatars#51]
Hello, thanks so far for this plugin! I'm running into the following issue:

At the moment, I have a function in the functions.php that is hooking the shortcode above the woocommerce_before_edit_account_form area within the WooCommerce Dashboard on the /edit-account/ page. 

So here's what happens...
1. When a user navigates to _mywebsite.com/my-account/edit-account/_ they will see the Basic User Avatars upload form. 
2. The user will then choose their image and click _Update Avatar_
3. The page (/my-account/edit-account/) will then refresh on submit and redirect to /my-account/ with no profile avatar updated. 

So in other words, the page refreshes back to the /my-account/ page on submit and does not complete the update. 

This form does work on "regular" pages during testing. It also works in the back-end of the dashboard if I am to update there. 

Any idea on how I can remedy this issue? This is pretty important, so looking for a quick solution. :) 

Thanks again guys!!!

**Comment by alexisteleco** [comment]
Exactly the same problem over here

**Comment by dineshinwk4wp** [comment]
Experiencing the same issue.

**Comment by andrewlimaza** [comment]
Please pre-patch your site with this fix (It should be included in an upcoming release) -> https://github.com/strangerstudios/Basic-User-Avatars/pull/53
